doc/releases/changelog-dev.md

```diff
@@ -16,8 +16,7 @@
   ```python
   import pennylane as qml
 
-  dev = qml.device("default.clifford", tableau=True)
-
+  dev = qml.device("default.clifford", tableau=True)  
```

Comment:
Why this is changed?

Reply:
This was a suggested change by @trbromley. I just committed the suggestion there.